app/__init__.py

```diff
@@ -35,12 +35,14 @@ def create_app(config_name):
         if not framework_data['slug'] in application.config.get('DM_FRAMEWORK_CONTENT_MAP', {}):
             if framework_data['framework'] == 'g-cloud':
                 if framework_data['status'] != 'expired':
-                    content_loader.load_manifest(framework_data['slug'], 'services', 'search_filters')
+                    content_loader.load_manifest(framework_data['slug'], 'services', 'services_search_filters')
                 # we need to be able to display old services, even on expired frameworks
                 content_loader.load_manifest(framework_data['slug'], 'services', 'display_service')
                 content_loader.load_manifest(framework_data['slug'], 'services', 'download_results')
             elif framework_data['framework'] == 'digital-outcomes-and-specialists':
                 content_loader.load_manifest(framework_data['slug'], 'briefs', 'display_brief')
+                if framework_data['slug'] == 'digital-outcomes-and-specialists-2':
```

Comment:
It'd be nice if there was some criteria we could use that would 'just work' assuming DOS3==DOS2. Maybe just check it's not DOS1, as a last resort.

Reply:
Have updated to use a helper to fetch the latests DOS framework. Should keep it up to date!